DllImportGenerator/DllImportGenerator/OptionsHelper.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Interop
+{
+    public static class OptionsHelper
+    {
+        public const string UseMarshalTypeOption = "build_property.DllImportGenerator_UseMarshalType";
```

Comment:
If this can be `private` please mark them as such.

Reply:
These are public so the tests can access them.